resources/public/api/conf/2.0/application.raml

```diff
@@ -616,4 +616,56 @@ traits:
                         description: The specified arrival does not exist, has been archived or is not available to the EORI number.
                         value:
                           code: NOT_FOUND
+        /{messageId}:
+          get:
+            displayName: Get a cached message related to a arrival and message ID
+            description: |
+              Retrieve all messages related to a specific arrival and message ID that have been received within the past 31 days.
+
+              For example, this could include messages about route diversions or unloading permissions.
+
+              Movements older than 31 days and their related messages are archived and cannot be returned by the API.
+
+               The response can be returned in one of the following formats based on the provided `Accept` header:
 
+               **Note:** You can view a transit movement only if your Economic Operators Registration and Identification (EORI) number matches the EORI number that created the transit declaration for that movement. Your EORI number is part of the bearer token that you use for API authentication.
+            is:
+              - acceptSingleMessageGetHeader
+              - acceptHeaderInvalid
+            (annotations.scope): "common-transit-convention-traders"
+            securedBy: [ sec.oauth_2_0: { scopes: [ "common-transit-convention-traders" ] } ]
+            responses:
+              200:
+                body:
+                  application/json:
+                    description: JSON payload
+                    example: |
+                      {
+                        "_links":{
+                          "self":{
+                            "href": "/customs/transits/movements/arrivals/62f4ebbbf581d4aa/messages/62f4ebbb765ba8c2"
+                          },
+                          "arrival":{
+                            "href": "/customs/transits/movements/arrivals/62f4ebbbf581d4aa"
+                          }
+                        },
+                        "id": "62f4ebbb765ba8c2",
+                        "arrivalId": "62f4ebbbf581d4aa",
+                        "received": "2022-08-11T11:44:59.83705",
+                        "type": "IE007",
+                        "body":{
+                          "n1.CC007C": {
+                            "@PhaseID": "NCTS5.0",
+                            ...
+                          }
+                        }
```

Comment:
This bit isn't returning Json, it's returning XML. It should be something like:

```
body: "<ncts:CC007C PhaseID=\"NCTS5.0\" xmlns:ncts=\"http://ncts.dgtaxud.ec\">...</ncts:CC007C>"
```

Reply:
@drnaylor-oc I think the XML body is already present under the json wrapped xml header application/json+xml. As per my comment above I think this should be removed until we implement the json header

Reply:
Oh I see what happened... @prashanthmamidi you removed the wrong section - should have removed the json rather than json wrapped xml https://github.com/hmrc/common-transit-convention-traders/pull/338/commits/5750b54a8b17f8f07cd673d626715f1e33af9654

Reply:
It's not, though I don't know if that was because it was removed before I looked at it.

The problem here is that `application/json+xml` isn't actually techincally a MIME type that is returned, `application/json` is -- what we have here is basically Json where we say XML is embedded as a string, so in the context of the response, this is correct. It's what I did on the equivalent endpoint for returning bot the Json and XML+Json endpoints.

It's not great, but it is what it is.

Reply:
@prashanthmamidi and this should have stayed there:
* `application/vnd.hmrc.2.0+json-xml`: JSON-wrapped XML, where the message is provided as an XML string sent as part of JSON payload containing other metadata.

Reply:
(looks like we responded at the same time again...)

Reply:
@drnaylor-oc I see what you're saying. For departures, it does look like application/json+xml and I wonder if it's because we can't duplicate application/json?

 https://github.com/hmrc/common-transit-convention-traders/blob/main/resources/public/api/conf/2.0/application.raml#L364

Reply:
updated the raml